yarn.lock

```diff
@@ -1388,396 +1381,299 @@
   dependencies:
     "@opentelemetry/api" "^1.0.0"
 
-"@opentelemetry/api@^1.0.0", "@opentelemetry/api@^1.4.0", "@opentelemetry/api@^1.4.1":
+"@opentelemetry/api@^1.0.0", "@opentelemetry/api@^1.4.0":
```

Comment:
hmm, still duped. running `yarn-deduplicate` locally does remove it


the semver range doesn't lock things down, so won't affect consumers. but still 😀

Reply:
Yes no that I could fix the issue regrading the data format in regards to timings I'll run `yarn-deduplicate` again, test the demo and push the change if everything is  working well 😃